src/module/ovs/bpf/user_op_exec.bpf.c

```diff
@@ -5,16 +5,21 @@
 
 /* Hook for usdt:dpif_netlink_operate__::op_flow_execute. */
 DEFINE_USDT_HOOK (
-	int res;
-	struct ovs_operation_event *exec_event =
-		get_event_zsection(event, COLLECTOR_OVS, OVS_OPERATION,
-				   sizeof(*exec_event));
-	if (!exec_event)
-		return 0;
+	struct ovs_operation_event *op;
+	batch_process_op(OVS_OP_EXEC, event, &op);
```

Comment:
If batch_process_op fails and we don't have batch information we don't know if the packet was filtered or not. Plus we don't have a OVS_OP_EXEC event.

In that case, I don't think it makes a lot of sense to add an entry to the tracking map. If we do, the ovs_do_execute_action tracepoint will emit tracking information but we won't be able to correlate it with anything with the it's userspace counterpart. It could be that somewhere back we do have that same queue id in some other event (maybe OVS_RECV_UPCALL, or maybe not because it's filtered). Either way, we could be creating misleading information.

I think ideally we should report this error so that post-processing functions take the information with caution.